lib/watir-ng/version.rb

```diff
@@ -1,4 +1,4 @@
 module WatirNg
   # :nodoc:
-  VERSION = "2.0.0"
+  VERSION = "2.0.1"
```

Comment:
I'd prefer it changed to `2.1.0`. Especially since this change makes `watir-ng` incompatible with Ruby 1.9.3. The `selenium-webdriver` dependency that `watir` has requires Ruby >= 2.0. 

This is why your build is failing in Travis.

Considering 1.9.3 should be long dead by now, I'm cool with that. But because a lot of testing organizations are slow to adopt, I'd like to keep a clean demarcation between works with 1.9.3 and doesn't. So `2.1.0` can be that demarcation.